tests/functional/includes/date.mock.js

```diff
@@ -0,0 +1,14 @@
+// Highjack the Date object so that our tests don't suddenly start failing...
```

Comment:
Is the issue that we need a constant date value? I'd rather we provide another mechanism for that than overriding the built-in since this could materially change the runtime.